deploy/database/data.button.sql

```diff
@@ -1152,3 +1152,315 @@ UPDATE button
+UPDATE button SET flavor_text="Grist is an avid student of Sun Tzu who can move objects with his arms and legs. He likes fast cars, manicures, and beating people up." WHERE name="Grist";
+UPDATE button SET flavor_text="Jellybean is a prominent patent attorney who can think with his spines. He likes collie-spaniel crosses, red ink, and beating people up." WHERE name="Jellybean";
+UPDATE button SET flavor_text="Lucky's father was murdered by an inconsiderate house guest. She likes road movies, vengeance, and beating people up." WHERE name="Lucky";
+UPDATE button SET flavor_text="Peace lives in the exile on the island of St Helena, where she enjoys swing dance, solitaire, and beating people up." WHERE name="Peace";
+UPDATE button SET flavor_text="Reaver takes unparalleled pleasure in bringing joy to her fellow beings. She enjoys surprise gift giving, platonic love, and beating people up." WHERE name="Reaver";
+UPDATE button SET flavor_text="Shepherd has made ignorance of obvious things his lifelong ambition. He likes spoons, forgetting how to do stuff, and beating people up." WHERE name="Shepherd";
+UPDATE button SET flavor_text="Strik becomes more accustomed to his new lifestyle with each passing day. He likes quiet walks in the moonlight, bashing things with his head, and beating people up." WHERE name="Strik";
+UPDATE button SET flavor_text="Wastenott is dead, dead, dead, dead. He likes getting out once in a while, talking to old friends and beating people up." WHERE name="Wastenott";
+UPDATE button SET flavor_text="Al-Khwarizmi wrote an amazing book about solving everyday problems in life and business with mathematics. Our word \"algebra\" came from the title of his book, and the term \"algorithm\" is from a Latin translation of his name." WHERE name="Al-Khwarizmi";
+UPDATE button SET flavor_text="Best known for his Theory of Numbers, Carl Friedrich Gauss startled his grade school teacher by deducing that the sum of the integers from 1 to 100 was really the sum of 50 pairs of integers which all add to 101." WHERE name="Carl Friedrich Gauss";
+UPDATE button SET flavor_text="Before becoming the Akami Professor of Internet Mathematics at UC San Diego, Dr. Fan Chung Graham directed math research labs at Bellcore. She and her husband Ron developed semi-random routing theories that keep the Internet running." WHERE name="Fan Chung Graham";
+UPDATE button SET flavor_text="Leonard Eugene Dickson loved pure mathematics. He wrote 267 research papers and 18 books, including the three volume \"History of the Theory of Numbers.\" His favorite quote: \"Thank God that number theory is unsullied by any applications.\"" WHERE name="Leonard Eugene Dickson";
+UPDATE button SET flavor_text="One of the best known and most respected mathematicians of the 20th century, Paul Erdos (air-dish) wrote or co-authored 1,475 papers. As he put it, \"A mathematician is a machine for turning coffee into theorems.\"" WHERE name="Paul Erdos";
```

Comment:
As mentioned on IRC, the o in Erdos has a Hungarumlaut on it.  If you respond to this comment, you too can type Hungarumlaut.